.circleci/config.yml

```diff
@@ -425,6 +426,21 @@ jobs:
           name: Update RPM repository
           command: './scripts/build/update_repo/update-rpm.sh "oss" "$GPG_KEY_PASSWORD" "$CIRCLE_TAG" "dist"'
 
+  build-oss-msi:
+    docker:
+      - image: grafana/grafana-oss-ci-msi-build
```

Comment:
I would like to see this PR include everything needed to build this image. Without it becomes very hard to debug when something goes wrong. I have split out Docker images previously and I'm trying to avoid that now, especially when it comes to the scripts that are.

Reply:
I can do that, the scripts needed are self-contained, it just has to run inside a wixtoolset container for the process to work.